### PR TITLE
feature/implementation

### PR DESCRIPTION
**Pull request by decanus**

### CODE REVIEW

**Comment by decanus** [comment]
@Arachnid I'm in favour of keeping the Blacklist contract itself as simple as possible.

File: contracts/Blacklist.sol [hunk 1]
```diff
@@ -0,0 +1,33 @@
+pragma solidity ^0.4.17;
+
+import "./BlacklistInterface.sol";
+
+contract Blacklist is BlacklistInterface {
+
+    address public owner;
+
+    mapping (bytes32 => bool) public blacklisted;
```
Comment (Arachnid) [inline contracts/Blacklist.sol]: Either this should be private, or `isBlacklisted` should be omitted, since they both serve the same purpose.

Comment (decanus) [reply contracts/Blacklist.sol]: @Arachnid removed the public part

**Review by Arachnid — APPROVED** [review]
Should probably add a way to change ownership.

**Pull request merged**